examples/base-ui-nextjs-tailwind-ts/next.config.mjs

```diff
@@ -1,4 +1,4 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {};
 
-module.exports = nextConfig;
+export default nextConfig;
```

Comment:
ESM should be the default at this stage, no? It's not what `npx create-next-app@latest` outputs but I don't see why it wouldn't make more sense.